### ISSUE TITLE
Add the ability of hiding apps in the app drawer

### ISSUE THREAD

**Issue by savethekiddes** [magneticchen/Daijishou#93]
There's a lot of unnecessary system apps on phones that you can't uninstall via traditional means, and I feal that using solutions like Island or adb is a little too invasive for just hiding from the apps drawer. So, please, add a dialog to hide apps from the Apps menu.

**Comment by magneticchen** [comment]
Please organize your requests. Your requests have exceed certain amounts in a short period and without proper documentation and trials and errors first (Although some of them cannot be done by yourself). Sometime it feels more like opinions compare then articulate suggestions. Although some of them are doable but is not in a shape of a good software design pattern and require some knowledge to understand how and why it should or shouldn't. There are some good examples for contributing. For instance, platforms and players, that some users making their own platforms, creating and fixing wallpaper packs by themself. Then did pull requests, merged into the repository. 

While this page does open for crowds as a space to submit suggestions. But if having **lots of** requests without organizing and structural preliminaries. I will suggest to join and initial your opinions in our [discord server](https://discord.com/invite/nJbxdT3QQE). It helps to refine opinions, construct the idea and evolve into a more matured state.

If you wish to further engaging. Hers is a good [guideline](https://allthecode.co/blog/post/how-to-ask-good-questions-as-a-developer) to read.

**Comment by savethekiddes** [comment]
You are totally right, and I'm sorry. I'm arleady doing a custom flash platform as of now. But I can't modify the app drawer itself as the app is colsed source.

**Comment by magneticchen** [comment]
> But I can't modify the app drawer itself as the app is closed source.
Yes. And it is actually a proper idea. But you know, opinions are from everywhere. 🙂 I have to organize it a bit.

**Comment by magneticchen** [comment]
Requests are welcome. But have reach the level that hard to handle. 🥵 And btw I am taking a break. Some of your request might able to organize into the 1.4 tho.

**Comment by savethekiddes** [comment]
Oke. Thank you for your collaboration and making this project grow.

**Comment by magneticchen** [comment]
Thanks for the understanding. 🙏
